packages/yourloops/test/integration/assert/monitoring-alerts-parameters.assert.ts

```diff
@@ -184,6 +184,75 @@ export const checkMonitoringAlertsForPatientSaveButtonMmol = async (): Promise<v
   expect(screen.getByText('Patient update succeeded')).toBeVisible()
 }
 
+export const checkMonitoringAlertsForPatientDiscardButton = async (): Promise<void> => {
+  const monitoringAlertsSection = within(screen.getByTestId('monitoring-alerts-configuration-section'))
+
+  const lowBgInput = monitoringAlertsSection.getByRole('spinbutton', { name: 'Low blood glucose input' })
+  const highBgInput = monitoringAlertsSection.getByRole('spinbutton', { name: 'High blood glucose input' })
+  const veryLowBgInput = monitoringAlertsSection.getByRole('spinbutton', { name: 'Very low blood glucose input' })
+  const outOfRangeThreshold = monitoringAlertsSection.getByTestId('basic-dropdown-out-of-range-selector')
+  const hypoThreshold = monitoringAlertsSection.getByTestId('basic-dropdown-hypo-threshold-selector')
+  const nonDataTxThreshold = monitoringAlertsSection.getByTestId('basic-dropdown-non-data-selector')
+  const discardButton = monitoringAlertsSection.getByRole('button', { name: 'Discard changes' })
+
+  expect(within(monitoringAlertsSection.getByTestId('low-bg-text-field-id')).getByText('mmol/L')).toBeVisible()
+  expect(within(monitoringAlertsSection.getByTestId('high-bg-text-field-id')).getByText('mmol/L')).toBeVisible()
+  expect(within(monitoringAlertsSection.getByTestId('very-low-bg-text-field-id')).getByText('mmol/L')).toBeVisible()
+  expect(lowBgInput).toHaveValue(2.8)
+  expect(highBgInput).toHaveValue(7.8)
+  expect(veryLowBgInput).toHaveValue(2.2)
+  expect(within(outOfRangeThreshold).getByRole('combobox')).toHaveTextContent('5%')
+  expect(within(hypoThreshold).getByRole('combobox')).toHaveTextContent('10%')
+  expect(within(nonDataTxThreshold).getByRole('combobox')).toHaveTextContent('15%')
+
+  expect(discardButton).toBeEnabled()
+
+  await userEvent.clear(lowBgInput)
+  await userEvent.type(lowBgInput, '3.55')
+  expect(within(monitoringAlertsSection.getByTestId('low-bg-text-field-id')).getByText('Value must be a number')).toBeInTheDocument()
+  expect(discardButton).toBeEnabled()
+  await userEvent.clear(lowBgInput)
+  await userEvent.type(lowBgInput, '4.8')
+  expect(discardButton).toBeEnabled()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is it really relevant to check whether the discard button is enabled, given that there is no condition in the code to disable it?

Reply:
No, it does not bring any value :+1: